app/helpers/patients_helper.rb

```diff
@@ -14,6 +14,10 @@ def race_ethnicity_options
      'Native American', 'Mixed Race/Ethnicity', 'Other']
   end
 
+  def language
+    [nil, 'English', 'Spanish', 'French', 'Korean']
```

Comment:
I think I'd rather store the nil instead of English for people who prefer english, since that's the bulk of our client base last I knew. I think that buys us a little bit of extra data space in that we won't have to save a default string 5000 times.

tldr: Can we remove the English option here, or better yet, have it display as 'English' but have the value of nil? I think that's the best of both worlds but if it winds up being a hassle, nbd.

fyi @alisonnjones -- please raise a flag NLT Wednesday afternoon if you'd prefer a different course of action.